components/style/invalidation/element/invalidator.rs

```diff
@@ -923,6 +931,64 @@ impl<'a, 'b: 'a, E> InvalidationCollector<'a, 'b, E>
         );
     }
 
+    /// Check whether a dependency should be taken into account, using a given
+    /// visited handling mode.
+    fn check_dependency(
+        &mut self,
+        visited_handling_mode: VisitedHandlingMode,
+        dependency: &Dependency,
+        relevant_link_found: &mut bool,
```

Comment:
Why does this need to be an outparam at all? It doesn't appear to be used in the caller.